src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Startup.cs

```diff
@@ -152,6 +154,14 @@ public void ConfigureServices(IServiceCollection services)
         services.AddApiVersioning().AddMvc().AddApiExplorer();
         services.AddEndpointsApiExplorer();
 
+        // Swagger
+
+        if (!hostEnvironment.IsIntegrationTest())
```

Comment:
I'm guessing that the need to move all of these things is that `Program.cs` is only invoked when fully starting the service in a hosted environment, whereas the Swashbuckle CLI need only fire up the DI container?

Reply:
I _think_ the Swashbuckle CLI can technically detect `Program` and run that (e.g. for minimal APIs), but because we have both `Program` and `Startup`, it seems to preferentially detect `Startup` and boots up an application using that. Consequently none of the Swagger config was being picked up when it was in `Program`, hence having to move it.

This has helped me realise though that this boolean was supposed to be `if (_appOptions.EnableSwagger)`, so I've changed that now :smile: